boards/kernel_layout.ld

```diff
@@ -191,6 +191,8 @@ SECTIONS
         . = ALIGN(4);
         _sapps = .;
 
+	LONG(0)
```

Comment:
I think we need a comment on this so that future linker script editors know why it is there.

Reply:
Also, does this break the bundling mechanism right now? I think this needs to come after the `KEEP(*(.app*))` below

Reply:
> Also, does this break the bundling mechanism right now? I think this needs to
> come after the `KEEP(*(.app*))` below

Okay I'll document the reason for `LONG(0)` and move it after `KEEP(*(.app*))`
in the next push.

Btw, where are we generating objects with `.app*` section which requires us to
have `KEEP(*(.app*))`?